src/components/Announcements/AnnouncementsList.vue

```diff
@@ -1,10 +1,19 @@
 <template>
   <div class="scroll-container announcements-container" v-bind:style="{ 'maxHeight': '20rem' }">
-      <div v-for="a in announcements" v-bind:key="a.id" class="announcement-wrapper">
-        <h3>{{a.title}}</h3>
-        <p>{{ toStringDate(a.created) }}</p>
-        <p>{{a.description}}</p>
+    <div v-if="announcements.length === 0">
+      <div class="blank-card">
+        <p>There is nothing to be announced!</p>
       </div>
+    </div>
+    <div v-else v-for="a in announcements" v-bind:key="a.id" @click="showAnnouncement(a)" >
+      <div class="announcement-card">
+        <div class="announce-header">
+          <div class="announce-title">{{a.title}}</div>
+          <div class="announce-date">{{ toStringDate(a.created) }}</div>
```

Comment:
It's typically better to use computed properties for content rendered. This is because the function is only called when a.created changes, where as here, the function is called everytime the anything on the page changes (I think). Doesn't really matter for our project, but fun vue fact!

Reply:
👍

Reply:
Does that work with this in a for loop?